recipes/r-gggenomes/meta.yaml

```
@@ -0,0 +1,110 @@
+{% set version = '1.0.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-gggenomes
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/gggenomes_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/gggenomes/gggenomes_{{ version }}.tar.gz
+  sha256: 3cdee64dc9156a0cb3eb5d3bb8b83df751ced28b2b84cc1d545fafda179f7538
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=4.2
+    - r-colorspace
+    - r-dplyr
+    - r-ellipsis
+    - r-ggplot2 >=3.5.0
+    - r-jsonlite
+    - r-magrittr
+    - r-purrr
+    - r-readr >=2.0.0
+    - r-rlang
+    - r-scales
+    - r-snakecase
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-tidyselect
+    - r-vctrs
+  run:
+    - r-base >=4.2
+    - r-colorspace
+    - r-dplyr
+    - r-ellipsis
+    - r-ggplot2 >=3.5.0
+    - r-jsonlite
+    - r-magrittr
+    - r-purrr
+    - r-readr >=2.0.0
+    - r-rlang
+    - r-scales
+    - r-snakecase
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-tidyselect
+    - r-vctrs
+
+test:
+  commands:
+    - $R -e "library('gggenomes')"           # [not win]
+    - "\"%R%\" -e \"library('gggenomes')\""  # [win]
+
+about:
+  home: https://github.com/thackl/gggenomes
+  license: MIT
+  summary: An extension of 'ggplot2' for creating complex genomic maps. It builds on the power
+    of 'ggplot2' and 'tidyverse' adding new 'ggplot2'-style geoms & positions and 'dplyr'-style
+    verbs to manipulate the underlying data. It implements a layout concept inspired
+    by 'ggraph' and introduces tracks to bring tidiness to the mess that is genomics
+    data.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - DonatWulf

```

Please retain the CF R team (the linter complaint can be ignored):

```suggestion
    - conda-forge/r
    - DonatWulf
```